experiments/obp_connector/run_real.py

```diff
@@ -0,0 +1,185 @@
+        "Unifrom Random"
+    ]
+    action_dist_list = [
+        action_dist_ipw_lr,
+        action_dist_ipw_rf,
+        action_dist_random
+    ]
+
+    ope = OffPolicyEvaluation(
+        bandit_feedback=bandit_feedback_test,
+        ope_estimators=[InverseProbabilityWeighting()]
+    )
+
+    for name, action_dist in zip(policy_names, action_dist_list):
+        estimated_policy_value = ope.estimate_policy_values(
+            action_dist=action_dist,
+        )
+
+        print(f'policy value of {name}: {estimated_policy_value}')
+
+
+def main(_):
+    args = FLAGS.config
+
+    spark = State(get_spark_session(**args.spark_params)).session
+    spark.sparkContext.setLogLevel('ERROR')
+
+    logger = get_logger("replay", logging.INFO)
+
+    dataset = OpenBanditDataset(behavior_policy=args.behavior_policy, data_path=args.data_path, campaign='all')
+    bandit_feedback_train, bandit_feedback_test = dataset.obtain_batch_bandit_feedback(test_size=args.test_size, is_timeseries_split=True)
+
+    on_policy_policy_value_random = OpenBanditDataset.calc_on_policy_policy_value_estimate(behavior_policy='random',
+                                                                                    campaign='all',
+                                                                                    data_path=args.data_path,
+                                                                                    test_size=args.test_size,
+                                                                                    is_timeseries_split=True)
+
+    on_policy_policy_value_bts = OpenBanditDataset.calc_on_policy_policy_value_estimate(behavior_policy='bts',
+                                                                                    campaign='all',
+                                                                                    data_path=args.data_path,
+                                                                                    test_size=args.test_size,
+                                                                                    is_timeseries_split=True)
+
+    # eval_baselines(dataset, bandit_feedback_train, bandit_feedback_test)
+
+    model = globals()[args.model](**args.params)
+
+    learner = RePlayOfflinePolicyLearner(n_actions=dataset.n_actions,
+                                         replay_model=model,
+                                         len_list=dataset.len_list,)
+
+    if args.opt.do_opt:
+        opt_params = args.opt_params
+        bandit_feedback_subset = bandit_subset(opt_params.subset_borders, bandit_feedback_train)
+        logger.info(learner.optimize(bandit_feedback_subset, opt_params.val_size, param_borders=args.opt.param_borders, budget=opt_params.budget))
+
+    timestamp = None
+    if "timestamp" in bandit_feedback_train.keys():
+        timestamp = bandit_feedback_train["timestamp"]
+    else:
+        timestamp = np.arange(bandit_feedback_train["n_rounds"])
```

Comment:
Now usage of new stuff is a little bit code-consuming. 
For example timestamp preparation part seems to be equal in every run. What about moving it to `.fit()` (e.g. pass raw `bandit_feedback_train` to `.fit()`)?